### PR TITLE
Add orange3-chem recipe

### PR DESCRIPTION
**Pull request by mat-cor**

### CODE REVIEW

File: recipes/example/meta.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@
 # Jinja variables help maintain the recipe as you'll update the version only here.
 {% set name = "simplejson" %}
 {% set version = "3.8.2" %}
-{% set sha256 = "d58439c548433adcda98e695be53e526ba940a4b9c44fb9a05d92cd495cdd47f" %}
+{% set sha256 = "b29f3cbab8881b049a43ba44ed2fed68f9e1f6862496199d884091052e6a3d1d" %}
```
Comment (CJ-Wright) [inline recipes/example/meta.yaml]: please reset this SHA

File: recipes/orange3-chem/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    - python
+    - orange3 >=3.4.3
+    - pandas
+    - numpy
+    - scipy >=0.17
+    - rdkit
+
+test:
+  imports:
+    - orangecontrib.chem
```
Comment (CJ-Wright) [inline recipes/orange3-chem/meta.yaml]: Would you mind adding some more import tests?

File: recipes/orange3-chem/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+
```
Comment (CJ-Wright) [inline recipes/orange3-chem/meta.yaml]: Please remove extra line

File: recipes/orange3-chem/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
```
Comment (dougalsutherland) [inline recipes/orange3-chem/meta.yaml]: You'll need to add `preserve_egg_dir: True` for `orange3` to be able to pick this up.

File: recipes/orange3-chem/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - orange3 >=3.4.3
+    - pandas
+    - numpy
```
Comment (dougalsutherland) [inline recipes/orange3-chem/meta.yaml]: `setup.py` declares itself as needing `numpy >=1.10.0`.

File: recipes/orange3-chem/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - orange3 >=3.4.3
+    - pandas
```
Comment (dougalsutherland) [inline recipes/orange3-chem/meta.yaml]: Not declared in `setup.py`, and doesn't seem to be used...?

File: recipes/orange3-chem/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+    - numpy
+    - scipy >=0.17
```
Comment (dougalsutherland) [inline recipes/orange3-chem/meta.yaml]: Not declared in `setup.py`, and doesn't seem to be used...?

File: recipes/orange3-chem/meta.yaml [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+    - rdkit
```
Comment (dougalsutherland) [inline recipes/orange3-chem/meta.yaml]: Not declared in `setup.py`, though it is used.

Comment (mat-cor) [reply recipes/orange3-chem/meta.yaml]: RDKit package it's not pip-installable, README specifies to install it before installing the add-on

File: recipes/orange3-chem/meta.yaml [hunk 9]
```diff
@@ -0,0 +1,47 @@
...
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [py2k]
+  skip: True  # [win]
```
Comment (bgruening) [inline recipes/orange3-chem/meta.yaml]: If you want to skip both you need to use: `[py2k or win]`. However, the tests are passing to I'm wondering if the skip for py2k is needed or if your tests are enough to catch this?

Comment (dougalsutherland) [reply recipes/orange3-chem/meta.yaml]: I think this actually works for conda-build, it just makes some ways of reading it (maybe the linter?) spit out errors.

**Pull request closed without merging**